pkg/api/v1/resource/helpers.go

```diff
@@ -122,6 +122,11 @@ func GetResourceRequestQuantity(pod *v1.Pod, resourceName v1.ResourceName) resou
 		return requestQuantity
 	}
 
+	if resourceName == v1.ResourceSwap {
+		//TODO (pacoxu) currently, pod swap setting is not supported
+		return requestQuantity
+	}
```

Comment:
Isn't this covered by the default case on L117? Neither swap nor PIDs can be set in the pod spec iirc.

Reply:
Not sure if there is a possibility to implement the support for pod swap limit?
If no plan, the todo can be removed.